mappings/net/minecraft/client/gui/screen/Bans.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_tkoofxmf net/minecraft/client/gui/screen/Bans
```

Comment:
Needs a more descriptive name such as `BanScreens`

Reply:
Updated to `BanScreen`.